### PR TITLE
Update framework versions to v3 and fix runtimes

### PR DESCRIPTION
**Pull request by meddy672**
I have updated the frameworkVersion to v3 and fixed the runtimes issues.

### CODE REVIEW

File: yarn.lock
Comment (medikoo) [inline yarn.lock:1]: Let's not introduce this file

**Review by medikoo — CHANGES_REQUESTED** [review]
@meddy672 thank you! It looks great, there's just one file in the PR which shouldn't be there

File: yarn.lock
Comment (meddy672) [reply yarn.lock:1]: > Let's not introduce this file

I have removed the `yarn.lock` file at the root directory level.

File: openwhisk-python-simple-http-endpoint/serverless.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 service: python-service
 
+frameworkVersion: "3"
+
 provider:
   name: openwhisk
-  runtime: python:3
+  runtime: python3.7
```
Comment (medikoo) [inline openwhisk-python-simple-http-endpoint/serverless.yml:7]: Why not `python3.9` ?

**Review by medikoo — COMMENTED** [review]
Thanks @meddy672 I have just one last question towards Python version

File: openwhisk-python-simple-http-endpoint/serverless.yml (hunk 1 above)
Comment (meddy672) [reply openwhisk-python-simple-http-endpoint/serverless.yml:7]: > Why not `python3.9` ?

Sure I can update python to v3.9. Fix should be up shortly.

**Pull request merged**